### PR TITLE
User-guide: add user-guide URL

### PR DESCRIPTION
**Pull request by fmuyassarov**
Although the URL is not the best https://metal3io.netlify.app/, add it for now rather than hiding the content while it is be being added, because it can already benefit some users. I'm checking with CNCF and @russellb to have an alias to redirect the content to https://book.metal3.io.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,11 @@ Kubernetes interfaces.
 * [Quick start](http://metal3.io/try-it.html)
 * [Demos](https://www.youtube.com/watch?v=VFbIHc3NbJo&list=PL2h5ikWC8viKmhbXHo1epPelGdCkVlF16&ab_channel=Metal3)
 * [Blog posts](https://metal3.io/blog/index.html)
-* [FAQ](https://metal3.io/blog/index.html)
+
+## Documentation
+
+Please see our [user-guide](https://metal3io.netlify.app/) to familiarize yourself with Metal3 features. We are currently in the process of writing
```
Comment (furkatgofurov7) [inline README.md:20]:
```suggestion
Please see our [user-guide](https://metal3io.netlify.app/) to familiarize yourself with Metal3 and its features. We are currently in the process of writing
```

File: README.md [hunk 2]
```diff
@@ -14,7 +14,11 @@ Kubernetes interfaces.
 * [Quick start](http://metal3.io/try-it.html)
 * [Demos](https://www.youtube.com/watch?v=VFbIHc3NbJo&list=PL2h5ikWC8viKmhbXHo1epPelGdCkVlF16&ab_channel=Metal3)
 * [Blog posts](https://metal3.io/blog/index.html)
-* [FAQ](https://metal3.io/blog/index.html)
```
Comment (furkatgofurov7) [inline README.md:17]: Is this change intentional?

Comment (fmuyassarov) [reply README.md:17]: yes, because it is not a FAQ but rather the same as ` [Blog posts](https://metal3.io/blog/index.html)`

File: README.md (hunk 1 above)
Comment (zaneb) [inline README.md:20]:
```suggestion
Please see our [user-guide](https://metal3io.netlify.app/) to familiarize yourself with Metal³ features. We are currently in the process of writing
```

**Review by zaneb — COMMENTED** [review]
I just noticed this broken link today as well: https://github.com/metal3-io/metal3-docs/blob/main/docs/user-guide/README.md?plain=1#L33

File: README.md (hunk 2 above)
Comment (furkatgofurov7) [reply README.md:17]: That is most probably because there is no way to point to FAQ page directly on that page

**Comment by fmuyassarov** [comment]
> I just noticed this broken link today as well: https://github.com/metal3-io/metal3-docs/blob/main/docs/user-guide/README.md?plain=1#L33

fixed
